lib/wasi/src/runtime/resolver/utils.rs

```diff
@@ -56,13 +56,39 @@ pub(crate) fn http_error(response: &HttpResponse) -> Error {
     Error::msg(status)
 }
 
+pub(crate) fn file_path_from_url(url: &Url) -> Result<PathBuf, Error> {
+    debug_assert_eq!(url.scheme(), "file");
+
+    // Note: The Url::to_file_path() method is platform-specific
+    cfg_if::cfg_if! {
+        if #[cfg(any(unix, windows, target_os = "redox", target_os = "wasi"))] {
```

Comment:
redox?

Reply:
MacOS counts as `unix`. The `#[cfg]` is copied straight from the `Url::from_file_path()` function:

https://github.com/servo/rust-url/blob/206d37851b9313ef3a6ecb83766d9bbc65d466be/url/src/lib.rs#L2422-L2440